src/crimson/common/fatal_signal.h

```diff
@@ -3,12 +3,14 @@
 
 #pragma once
 
+#include <csignal>
```

Comment:
nit, how about forward declaring it instead, like
```c++
struct siginfo_t
```
?

Reply:
Already tried that; unfortunately, it's `typedef` :-(.